MekHQ/src/mekhq/gui/model/FinanceTableModel.java

```diff
@@ -61,34 +61,33 @@ public String getColumnName(int column) {
 
     public Object getValueAt(int row, int col) {
         Transaction transaction = getTransaction(row);
-        long amount = transaction.getAmount();
-        long balance = 0;
+        Money amount = transaction.getAmount();
+        Money balance = Money.zero();
         for(int i = 0; i <= row; i++) {
-            balance += getTransaction(i).getAmount();
+            balance = balance.plus(getTransaction(i).getAmount());
         }
-        DecimalFormat formatter = new DecimalFormat();
         if(col == COL_CATEGORY) {
             return transaction.getCategoryName();
         }
         if(col == COL_DESC) {
             return transaction.getDescription();
         }
         if(col == COL_DEBIT) {
-            if(amount < 0) {
-                return formatter.format(-1 * amount);
+            if(amount.isNegative()) {
+                return amount.multipliedBy(-1).toAmountAndSymbolString();
```

Comment:
if there was a getAbsoluteAmount() that would be nicer.

Reply:
Good idea.